src/WebJobs.Script.WebHost/Models/MSISpecializationPayload.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    public class MSISpecializationPayload
```

Comment:
nit: perhaps name MSIContext to match the Context naming we use?

Reply:
Fixed